bubuku/features/remote_exec.py

```diff
@@ -5,16 +5,18 @@
 from bubuku.features.migrate import MigrationChange
 from bubuku.features.rebalance import RebalanceChange
 from bubuku.features.restart_on_zk_change import RestartBrokerChange
+from bubuku.features.swap_partitions import SwapPartitionsChange, load_swap_data
 from bubuku.zookeeper import BukuExhibitor
 
 _LOG = logging.getLogger('bubuku.features.remote_exec')
 
 
 class RemoteCommandExecutorCheck(Check):
-    def __init__(self, zk: BukuExhibitor, broker_manager: BrokerManager):
+    def __init__(self, zk: BukuExhibitor, broker_manager: BrokerManager, api_port):
```

Comment:
For me this `api_port` property is strange. Bubuku instance can know only its own api port but not the others'. So I don't really see much sense to have it as a configured property, I would be fine with just a constant.